src/crimson/mgr/client.cc

```diff
@@ -168,6 +170,26 @@ void Client::report()
   });
 }
 
+void Client::_send_report()
+{
+  gate.dispatch_in_background(__func__, *this, [this] {
+  if (!conn) {
+    logger().warn("cannot send report; no conn available");
+    return seastar::now();
+  }
+  auto report = make_message<MMgrReport>();
+  if (daemon_name.size()) {
+    report->daemon_name = daemon_name;
+  } else {
+    report->daemon_name = local_conf()->name.get_id();
+  }
+  report->service_name = service_name;
```

Comment:
service_name will always remain empty for now, although it seems that handle_report expects that.